### ISSUE TITLE
Bar no longer shows up for me

### ISSUE THREAD

**Issue by ms3056** [karstenpedersen/obsidian-daily-note-navbar#21]
Default theme, no snippets, no plugins. The bar is not showing up with the latest release. I tried enabling Periodic Notes (which I no longer use) but that didn't seem to trigger the bar's appearance either. 

![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/23712700/5ef29950-e211-4ca8-accc-5b7086eb5aca)

![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/23712700/87e9686f-89f9-4c27-9ec8-6973b6ae702a)

**Comment by karstenpedersen** [comment]
I am looking into it. Have you tried restarting Obsidian if you just changed them?

**Comment by karstenpedersen** [comment]
I don't get the issue, I have tried in my development vault, personal vault, and also in newly created vaults.

What platform/operating system are you on? I don't think it should matter, but just to check.

**Comment by QuesadaGrande** [comment]
Edit:
**update**
For some strange reason the daily note date format was different than the one in the daily note plugin. This was causing the navnar not showing up!!


I can confirm that the navbar no longer shows up. I have the same problem.
Latest obsidian installer on windows machine

**Comment by karstenpedersen** [comment]
> Edit: **update** For some strange reason the daily note date format was different than the one in the daily note plugin. This was causing the navnar not showing up!!
> 
> I can confirm that the navbar no longer shows up. I have the same problem. Latest obsidian installer on windows machine

What was the note format if you know it was different and were you able to fix it?

**Comment by QuesadaGrande** [comment]
> > Edit: **update** For some strange reason the daily note date format was different than the one in the daily note plugin. This was causing the navnar not showing up!!
> 
> > 
> 
> > I can confirm that the navbar no longer shows up. I have the same problem. Latest obsidian installer on windows machine
> 
> 
> 
> What was the note format if you know it was different and were you able to fix it?

The format I use for my daily note is: YYYY-MM-DD (dddd)

I changed this in the daily note navbar settings and everything works perfectly

**Comment by karstenpedersen** [comment]
> > > Edit: **update** For some strange reason the daily note date format was different than the one in the daily note plugin. This was causing the navnar not showing up!!
> > 
> > 
> > > 
> > 
> > 
> > > I can confirm that the navbar no longer shows up. I have the same problem. Latest obsidian installer on windows machine
> > 
> > 
> > What was the note format if you know it was different and were you able to fix it?
> 
> The format I use for my daily note is: YYYY-MM-DD (dddd)
> 
> I changed this in the daily note navbar settings and everything works perfectly

I'm glad it works! I will close this issue, but feel free to create new issues if anything else comes up.

**Comment by ms3056** [comment]
Still doesn't work for me - my Daily Note is YYYY-MM-DD and this matches the settings. Am I doing something wrong? Could it be because the PATH of where my Daily Note is located? 

![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/23712700/3aeefb97-aee1-473e-a62a-8bf0382453ca)

**Comment by karstenpedersen** [comment]
> Still doesn't work for me - my Daily Note is YYYY-MM-DD and this matches the settings. Am I doing something wrong? Could it be because the PATH of where my Daily Note is located?
> 
> ![image](https://private-user-images.githubusercontent.com/23712700/344462007-3aeefb97-aee1-473e-a62a-8bf0382453ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk3NDgyNDksIm5iZiI6MTcxOTc0Nzk0OSwicGF0aCI6Ii8yMzcxMjcwMC8zNDQ0NjIwMDctM2FlZWZiOTctYWVlMS00NzNlLWE2MmEtOGJmMDM4MjQ1M2NhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjMwVDExNDU0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIwY2IxNmFiYzU2YzUyYjMxNmY1ZThiNTNiOGNiNTBkYmY3NGRhOTlhMDNjNjUzY2Q1ZjE4NmRiZjgwNTZjNDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.b-RuQ5Zk7UlQrov03NzUEnPufiELTK4XFzq1dyw_yDg)

Can you try and reinstall the plugin? It should not be the path, as I just check against the basename of the file. (the name of the file without the extension).

**Comment by karstenpedersen** [comment]
Can you also check the console in the Inspector by pressing Ctrl+shift+i after you open a daily note.
![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/53978294/e2f0cb51-fe5c-4006-b4e7-f96c76e49c0a)

**Comment by ms3056** [comment]
I tried over and over - the 0.1.5 version (and earlier) is the only version that works for me. I did get it to work on a fresh vault - but I was also able to break it. I turned OFF the Tab Title Bar in the Appearance Settings, then turned it on, and I couldn't get the NavBar to work. I had to turn off the NavBar and reload the UI, then turn it back on and it came back. 

With 0.1.5 it "just works". 

I still cannot get it to work in my current vault, no matter what I do. I have tried 0 plugins, 0 snippets and the default theme. 

I guess if it is working for everyone else - then close it. For now I will stick with 0.1.5. 

Thanks for trying - it doesn't make sense, but I've tried everything.

**Comment by karstenpedersen** [comment]
I have not tried it with turning off the header bar, but I think it makes sense that the plugin doesn't like that and requires a reload. I will try and look at it throughout this week. Thanks for checking that 0.1.5 works. I will look at that.

You don't have any custom CSS snippets?

**Comment by ms3056** [comment]
> I have not tried it with turning off the header bar, but I think it makes sense that the plugin doesn't like that and requires a reload. I will try and look at it throughout this week. Thanks for checking that 0.1.5 works. I will look at that.
> 
> You don't have any custom CSS snippets?

Yeah - I tried turning all CSS snippets off, default theme, no other plugins. For some reason I just can't get it to work in my vault. 

It works on a new vault though. Weird - but for now I'll just stick with 0.1.5 as it does what I need.

**Comment by focusaurus** [comment]
I also have this issue. Can't get the navbar to show up. 🤷🏻

**Comment by xgoku1** [comment]
I had this issue.

My Daily Notes format included subfolders, like: `YYYY/MM-MMMM/dddd, DD-MM-YYYY`

Initially I copy-pasted this to Daily Note Navbar → Settings → Daily note date format. This didn't work.

I changed it to just `dddd, DD-MM-YYYY` and it worked.

Because how it works is by scanning the name of the active file, and shows the navbar if it matches the provided format.

There's a little user confusion because the Date Format field used in Daily Notes also allows for customizing the Path, sub-folders etc. whereas the Daily Note Date Format field of this addon's settings specifically wants the "actual" name of the daily note alone without sub-folder paths.

**Comment by karstenpedersen** [comment]
I will close this issue for now. Feel free to create a new issue if this is still relevant.
